tests/integration/features/istio/production/upgrade-suite/istio_upgrade.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Upgrade Istio
+
+  Scenario: Upgrade from current release to last version
```

Comment:
We would need to invoke git command in a separate process programmatically, so it seemed to be more straightforward to use bash for latest release deployment and make it a prerequisite. Also the same way for other tests the deployed controller is a prerequisite so it is not inconsistent. I think it would be good to make it a separate make target the same way as deploy is, eg. deploy-latest-release which will execute the script. It will be more obvious from Makefile perspective that this is a prerequisite in addition to comment.